src/test/java/org/folio/rest/impl/StorageTestSuite.java

```diff
@@ -45,6 +46,15 @@ public class StorageTestSuite {
   public static final Header URL_TO_HEADER = new Header("X-Okapi-Url-to", "http://localhost:" + port);
   private static TenantJob tenantJob;
 
+  public static EmbeddedKafkaCluster KAFKA_CLUSTER;
+  public static final String KAFKA_ENV_VALUE = "test-env";
+  private static final String KAFKA_HOST = "KAFKA_HOST";
+  private static final String KAFKA_PORT = "KAFKA_PORT";
+  private static final String KAFKA_ENV = "ENV";
+  private static final String OKAPI_URL_KEY = "OKAPI_URL";
+  public static final int MOCK_KAFKA_PORT = NetworkUtils.nextFreePort();
```

Comment:
would be good to add unit tests that checks that events are sent, similar as in orders-storage,
for example like in PurchaseOrderLinesApiTest

Reply:
Coverd by unit tests aswell